.github/workflows/build.yml

```diff
@@ -0,0 +1,71 @@
+name: build branch
+
+on: [push, pull_request]
+
+jobs:
+
+  test:
+    runs-on: ubuntu-latest
+    name: Check branch
+    steps:
+    - uses: actions/checkout@v2
+
+    # Install Node 12
+    - uses: actions/setup-node@v1
+      with:
+        node-version: 12
+
+    # Cache node modules
+    - uses: actions/cache@v2
+      env:
+        cache-name: cache-node-modules
+      with:
+        # npm cache files are stored in `~/.npm` on Linux/macOS
+        path: ~/.npm
+        key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.os }}-build-${{ env.cache-name }}-
+          ${{ runner.os }}-build-
+          ${{ runner.os }}-
+
+    # Install environment
+    - run: npm config set spin false
```

Comment:
I prefer block run step:

```
    - name: Install environment
      run: |
        npm config set spin false
        npm install -g npm@7
        npm install -g codecov
        npm --version
```

With this, we can also remove comments for `name` field.